stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/DecoratorsProvider.kt

```diff
@@ -0,0 +1,5 @@
+package io.getstream.chat.android.ui.messages.adapter.viewholder.decorator
+
+public interface DecoratorsProvider {
```

Comment:
I'd name this `DecoratorProvider`, even if it provides a list of them - sounds nicer